include/drivers/i2c.h

```diff
@@ -163,6 +163,8 @@ struct i2c_slave_config;
 
 typedef int (*i2c_api_configure_t)(const struct device *dev,
 				   uint32_t dev_config);
+typedef int (*i2c_api_get_config_t)(const struct device *dev,
+				    uint32_t *dev_config);
```

Comment:
Please add a full function comment, including describing what dev_config is

Reply:
It is done under i2c_config_get, as the other API functions.